kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/data/order/OrderRepository.java

```diff
@@ -118,6 +120,8 @@ public void onResponse(Order response) {
 
             @Override
             public void onFailure(Throwable t) {
+                removeCachedOpenOrderByID(orderID);
```

Comment:
these 2 lines can be moved to a clearOpenAndPendingOffersOrderByID - used more in 2 places

Reply:
i would like to keep it as it is, and not combine those two methods into one, it will confuse others.